packages/map-template/src/components/ChatWindow/ChatListItemLocation/ChatListItemLocation.scss

```
@@ -0,0 +1,122 @@
+        gap: 1rem;
+        background: none;
+        border: none;
+        padding: 0;
+        margin: 0;
+        text-align: left;
+        cursor: pointer;
+        color: inherit;
+        font: inherit;
+
+        &:focus-visible {
+            outline: 2px solid currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)